modules/RTCStats/RTCStats.ts

```diff
@@ -0,0 +1,189 @@
+import rtcstatsInit from '@jitsi/rtcstats/rtcstats';
+import traceInit from '@jitsi/rtcstats/trace-ws';
+import EventEmitter from 'events';
+
+import { 
+    CONFERENCE_JOINED, 
+    CONFERENCE_LEFT, 
+    CONFERENCE_UNIQUE_ID_SET 
+} from '../../JitsiConferenceEvents';
+import JitsiConference from '../../JitsiConference';
+
+const RTC_STATS_WC_DISCONNECTED = 'rtcstats_ws_disconnected';
+const RTC_STATS_PC_EVENT = 'rtstats_pc_event';
+
+/**
+ * Filter out RTCPeerConnection that are created by callstats.io.
+ *
+ * @param {*} config - Config object sent to the PC c'tor.
+ * @returns {boolean}
+ */
+function connectionFilter(config) {
+    if (config?.iceServers[0] && config.iceServers[0].urls) {
+        for (const iceUrl of config.iceServers[0].urls) {
+            if (iceUrl.includes('callstats.io')) {
+                return true;
+            }
+        }
+    }
+}
+
+/**
+ * RTCStats Singleton that is initialized only once for the lifetime of the app, subsequent calls to init will be ignored.
+ * Config and conference changes are handled by the start method.
+ */
+class RTCStats {
+    private _initialized: boolean = false;
+    private trace: any = null;
+    public events: EventEmitter = new EventEmitter();
+
+    /**
+     * RTCStats "proxies" WebRTC functions such as GUM and RTCPeerConnection by rewriting the global objects.
+     * The proxies will then send data to the rtcstats server via the trace object.
+     * The initialization procedure must be called once when lib-jitsi-meet is loaded.
+     * 
+     * @param initConfig initial config for rtcstats.
+     * @returns {void}
+     */
+    init(initConfig: any) {        
+        const {
+            analytics: {
+                rtcstatsUseLegacy: useLegacy = false,
```

Comment:
Does this make sense still?

Reply:
Seems like chrome removed legacy support as of 117, but rtcstats still has it as an option, I'll remove the option here and make a task for removing support from rtcstats client.

Reply:
Hold on :-) What does it really imply? Note we support Chrome >= 72.

Reply:
Welp, removing it wouldn't break older Chrome users, as standard stats have been available for a while, so we would simply be forcing clients to use standard.
But this requires some work on the rtcstats client as previously mentioned, I'll add a note to make sure that standard stats are available from Chrome >= 72.